File: day three /java/afternoon session/ParkingSlotTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

public class ParkingSlotTest{
    @Test
    public void testThatTheParkingSpaceIsEmpty(){
    int [] number = {0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0};
    String expected = "empty";
    String actual = ParkingSlot.CheckParkingSlot(number);
    assertEquals(expected, actual);
    } 

     @Test
    public void testThatTheParkingSpaceIsFilled(){
    int [] number = {1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1};
    String expected = "empty";
    String actual = ParkingSlot.CheckParkingSlot(number);
    assertEquals(expected, actual);
    } 
}

File: day three /javascript/afternoon session /parkingslotTest.js
const  {CheckParkingSlot, parkingSpace} = require('./parkingslot')

test("if parking space is empty", () => {

       let parkingSpace = {0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0} 
    
       expect(CheckParkingSlot(parkingSpace)).toBe("empty")

})

test("if parking space is empty", () => {

       let parkingSpace = {1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
    
       expect(CheckParkingSlot(parkingSpace)).toBe("occupied")

})
